app/assets/stylesheets/caffeine/admin/widgets/medium.css.scss

```diff
@@ -67,15 +58,9 @@ $buttons: (
           background-color: $colorBaseSecondApplication;
         }
 
-        @each $suffix, $fa-var in $buttons {
+        @each $suffix in $buttons-last {
           &.medium-editor-action-#{$suffix} {
-            b {
-              display: none;
-            }
-
-            &:before {
-              @include faIcon($fa-var);
-            }
+            border-right: $padding solid $colorBaseFirstApplication;
```

Comment:
Name of variable `colorBaseFirstApplication` should be written in all lowercase letters with hyphens instead of underscores